src/Bicep.Core.UnitTests/Assertions/MockRegistryAssertions.cs

```diff
@@ -64,13 +66,16 @@ private AndConstraint<MockRegistryAssertions> HaveModuleCore(string tag, Stream
 
                 var config = manifest.Config;
                 config.MediaType.Should().Be("application/vnd.ms.bicep.module.config.v1+json", "config media type should be correct");
-                config.Size.Should().Be(0, "config size should be empty");
 
                 this.Subject.Blobs.Should().ContainKey(config.Digest, "module config digest should exist");
                 if (this.Subject.Blobs.ContainsKey(config.Digest))
                 {
                     var configBytes = this.Subject.Blobs[config.Digest];
-                    configBytes.Bytes.Should().BeEmpty("module config blob should be empty");
+                    config.Size.Should().Be(configBytes.ToArray().Length, "Config size field should match config size");
+                    JsonDocument? configJson = null;
+                    var convertToJson = () => configJson = JsonDocument.Parse(configBytes.Text);
```

Comment:
`JsonDocument` should be disposed and the `JsonElement` created from Parse should be cloned. It is recommended to call `JsonElementFactory.CreateElement(configBytes.Text)` which already handles that.

Reply:
f0e4d56a9f0a331ca6ba9dece291d4d50a12bc27